gradle/libs.versions.toml

```diff
@@ -28,38 +31,43 @@ kotlinx-coroutines = "1.10.1"
 
 slf4j = "2.0.16"
 
+kotlin-compiler = "2.1.10"
+kotlin-compiler-k2 = "2.2.0-dev-326"
```

Comment:
Why do we need dev version of Kotlin compiler?

Reply:
We don't actually, fixed